build/cqs.tsv

```diff
@@ -168,16 +229,19 @@ org.slf4j:slf4j-api	1.7.5	compile
 org.slf4j:slf4j-log4j12	1.7.5	compile
 org.tukaani:xz	1.0	compile
 picocontainer:picocontainer	1.2	compile
+tomcat:jasper-compiler	5.5.23	compile
+tomcat:jasper-runtime	5.5.23	compile
+xmlenc:xmlenc	0.52	compile
 
 com.datastax.cassandra:cassandra-driver-core	3.0.0	provided
 com.datastax.cassandra:cassandra-driver-mapping	3.0.0	provided
 javax.media:jai_codec	1.1.3	provided
 javax.media:jai_core	1.1.3	provided
+org.apache.hadoop:hadoop-annotations	2.5.2	provided
```

Comment:
This *provided* Hadoop dependency and the remaining in this diff are covered by https://dev.eclipse.org/ipzilla/show_bug.cgi?id=8504.